src/tests/nativeaot/SmokeTests/Reflection/Reflection.csproj

```diff
@@ -10,7 +10,7 @@
     <NoWarn>$(NoWarn);IL3050</NoWarn>
 
     <!-- Look for MULTIMODULE_BUILD #define for the more specific incompatible parts -->
-    <CLRTestTargetUnsupported Condition="'$(IlcMultiModule)' == 'true'">true</CLRTestTargetUnsupported>
+    <CLRTestTargetUnsupported Condition="'$(IlcMultiModule)' == 'true' or '$(TargetsAppleMobile)' == 'true'">true</CLRTestTargetUnsupported>
```

Comment:
Why is this getting disabled? (Is there a issue number we should link to?)

Reply:
I was using the old bits and the MethodInvoker couldn't be resolved. However, with the new bits, it should have been resolved.